src/sortStudents.ts

```diff
@@ -1,16 +1,78 @@
 
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number [];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades'
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+export function sortStudents(
+  students: Student [],
+  sortBy: SortType,
+  order: SortOrder,
+):Student [] {
+  const studentsNew: Student [] = [...students];
 
-export function sortStudents(students, sortBy, order) {
-  // write your function
+  switch (sortBy) {
+    case 'age': {
```

Comment:
```suggestion
    case SortType.Age: {
```
it will help to avoid mistypes in strings